components/director/internal/domain/webhook/repository_test.go

```diff
@@ -235,7 +236,28 @@ func TestRepositoryCreateMany(t *testing.T) {
 		// WHEN
 		err := sut.CreateMany(ctx, given)
 		// THEN
-		require.EqualError(t, err, "while creating Webhook [URL: unlucky, Type: CONFIGURATION_CHANGED]: while converting model to entity: some error")
+		require.EqualError(t, err, "while creating many webhooks: while converting model to entity: some error")
+	})
+
+	t.Run(testCaseErrorOnDBCommunication, func(t *testing.T) {
+		// GIVEN
+		mockConverter := &automock.EntityConverter{}
+		defer mockConverter.AssertExpectations(t)
+
+		given := []*model.Webhook{{ID: "one", URL: "unlucky", Type: model.WebhookTypeConfigurationChanged}, {ID: "two"}, {ID: "three"}}
+		mockConverter.On("ToEntity", *given[0]).Return(webhook.Entity{ID: "one"}, nil)
+
+		db, dbMock := testdb.MockDatabase(t)
+		defer dbMock.AssertExpectations(t)
+
+		dbMock.ExpectExec(regexp.QuoteMeta(expectedInsert)).WillReturnError(givenError())
```

Comment:
It would be better to check what happens, when first item is added properly, and inserting second fails

Reply:
discussed, it does not introduce any value